docs/framework/release-notes/2024/04-april-security-and-quality-rollup.md

```diff
@@ -0,0 +1,103 @@
+# April 2024 Security and Quality Rollup
+
+_Released April 9, 2024_
+
+## Summary of What's New in this Release
+
+### Security improvements
+
+#### CVE-2024-21409 – Remote Code Execution Vulnerability
+
+This security update addresses a remote code execution vulnerability detailed in [CVE 2024-21409](https://portal.msrc.microsoft.com/en-US/security-guidance/advisory/CVE 2024-21409).
```

Comment:
This link doesn't work for me. Should it point here? https://msrc.microsoft.com/update-guide/vulnerability/CVE-2024-21409